Update color button stakeholderdetails

93Belen opened the pull request:
Fixes #1965 

**Since the Issue only required to change colors, I didn't change anything else in the style**

- I added a `gay variant` to the Button override file.
- I added this variant to the buttons needed
- I changed the color of the button Back to List



_I notice that the Figma files don't show the styles for hover effect or disabled state, I added the hover effect but the gray variant still need the disabled style._

Hover style added: 
![Screenshot 2024-01-24 at 6 26 56 AM](https://github.com/hackforla/food-oasis/assets/98275292/b38b3513-4fbf-48af-b195-b877edd16d05)

qiqicodes commented on client/src/theme/overrides/Button.js:
```diff
@@ -105,6 +105,28 @@ export default function Button(theme) {
+            fontSize: "0.8125rem",
+          },
+          "&:hover": {
+            backgroundColor: '#E1E1E1',
```
The background color should be "#E2E5EE"  instead of "#E1E1E1" according the Figma.

<img width="848" alt="Screenshot 2024-01-26 at 4 35 33 PM" src="https://github.com/hackforla/food-oasis/assets/68657634/a2daeb32-7599-4f74-88b1-087af1294546">

93Belen replied:
Omg thats so helpful, I didn't see that in the sigma and I made up the hover state! I wasn't to far from the real one haha

qiqicodes commented on client/src/theme/overrides/Button.js:
```diff
@@ -105,6 +105,28 @@ export default function Button(theme) {
+          "&:hover": {
+            backgroundColor: '#E1E1E1',
+          },
+          "&:active": {
```
The active state is not listed in Figma. @JohnHaoHuang, can you clarify the active state?

93Belen commented:
@qiqicodes Hey! I added the two changes you suggested :) Regarding the Active State, I changed the background color to the same one as for the hover state, cause that seems to be the pattern in the other styles. I think we can leave it as placeholder until they tell us otherwise.
I hope everything looks ok now, thanks!
